dlp/src/main/java/dlp/snippets/QuickStart.java

```diff
@@ -14,7 +14,9 @@
  * limitations under the License.
  */
 
-package com.example.dlp;
+package dlp.snippets;
```

Comment:
Ok, but not really the standard for either us or Java.

Reply:
We don't have guidance on what the snippet package should be (but this is the package the canonical snippet has been in for a while.)

What would you like to be?